_inc/client/plans/index.jsx

```diff
@@ -12,35 +13,204 @@ import PlanGrid from './plan-grid';
 import { SingleProductBackup } from './single-product-backup';
 import QueryProducts from 'components/data/query-products';
 import QuerySite from 'components/data/query-site';
-import { getUpgradeUrl } from 'state/initial-state';
-import { getSitePlan, isFetchingSiteData } from 'state/site';
+import ProductCard from 'components/product-card';
+import { getPlanClass } from 'lib/plans/constants';
+import { getSiteRawUrl, getUpgradeUrl } from 'state/initial-state';
 import { getProducts, isFetchingProducts } from 'state/products';
+import { getActiveSitePurchases, getSitePlan, isFetchingSiteData } from 'state/site';
 
 export class Plans extends React.Component {
+	getProductCardPropsForPurchase( purchase ) {
+		const { siteRawlUrl } = this.props;
+
+		const planClass = getPlanClass( purchase.product_slug );
+
+		switch ( planClass ) {
+			case 'is-daily-backup-plan':
+				return {
+					title: __( 'Jetpack Backup {{em}}Daily{{/em}}', {
+						components: {
+							em: <em />,
+						},
+					} ),
+					subtitle: __( 'Purchased %(purchaseDate)s', {
+						args: {
+							purchaseDate: moment( purchase.subscribedDate ).format( 'YYYY-MM-DD' ),
+						},
+					} ),
+					description: (
+						<>
+							<p>
+								{ __( '{{strong}}Looking for more?{{/strong}}', {
+									components: {
+										strong: <strong />,
+									},
+								} ) }
+							</p>
+							<p>
+								{ __(
+									'With Real-time backups, we save as you edit and you’ll get unlimited backup archives.'
+								) }
+							</p>
+						</>
+					),
+					purchase,
+					isCurrent: true,
+				};
+			case 'jetpack_backup_realtime':
+				return {
+					title: __( 'Jetpack Backup {{em}}Real-Time{{/em}}', {
+						components: {
+							em: <em />,
+						},
+					} ),
+					subtitle: __( 'Purchased %(purchaseDate)s', {
+						args: {
+							purchaseDate: moment( purchase.subscribedDate ).format( 'YYYY-MM-DD' ),
+						},
+					} ),
+					description: __(
+						'Always-on backups ensure you never lose your site. Your changes are saved as you edit and you have unlimited backup archives.'
+					),
+					purchase,
+					isCurrent: true,
+				};
+			case 'is-personal-plan':
+				return {
+					title: __( 'Jetpack Backup {{em}}Daily{{/em}}', {
+						components: {
+							em: <em />,
+						},
+					} ),
+					subtitle: __( 'Included in your {{planLink}}Personal Plan{{/planLink}}', {
+						components: {
+							planLink: <a href={ `/plans/my-plan/${ siteRawlUrl }` } />,
```

Comment:
Should it be a relative path here and below?

Reply:
I agree. This probably needs to be updated after the migration from Calypso.